src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -1655,7 +1655,8 @@ static void our_sofia_event_callback(nua_event_t event,
 	}
 
 	if (sip && (status == 401 || status == 407)) {
-		sofia_reg_handle_sip_r_challenge(status, phrase, nua, profile, nh, sofia_private, session, gateway, sip, de, tags);
+		// ENGDESK-37943 disable responding to challenges
+		//sofia_reg_handle_sip_r_challenge(status, phrase, nua, profile, nh, sofia_private, session, gateway, sip, de, tags);
```

Comment:
Might be good to add a log here for visibility and awareness

Reply:
This should be configurable via sofia profile instead of disabling it totally. Otherwise this will impact other service